src/select-histogram/types.ts

```diff
@@ -0,0 +1,16 @@
+import { HistogramSliderProps } from "../histogram-slider";
+import { TruncateType } from "../shared/dropdown-list-v2/types";
+import { DropdownAlignmentType } from "../shared/dropdown-wrapper";
+
+export interface SelectHistogramSliderProps {
```

Comment:
to also include the following optional props:
- `className`
- `readOnly`
- `onBlur` - can be typed without the event arg as `() => void`